scripting/src/main/java/org/infinispan/scripting/impl/DataType.java

```diff
@@ -0,0 +1,78 @@
+package org.infinispan.scripting.impl;
+
+import org.infinispan.commons.CacheException;
+import org.infinispan.commons.marshall.Marshaller;
+
+import java.io.UnsupportedEncodingException;
+import java.util.Optional;
+
+public enum DataType {
+   UTF8(new Utf8Transformer()),
```

Comment:
On the JIRA I mentioned that probably we should use the media type format as defined by IANA: http://www.iana.org/assignments/media-types/media-types.xhtml